activerecord/test/cases/adapters/mysql2/mysql2_adapter_test.rb

```diff
@@ -322,6 +324,94 @@ def test_database_timezone_changes_synced_to_connection
     end
   end
 
+  def test_ignores_warnings_when_behaviour_ignore
+    old_action = ActiveRecord.db_warnings_action
+    ActiveRecord.db_warnings_action = :ignore
+    @conn.execute('SELECT 1 + "foo"')
```

Comment:
This test isn't asserting anything, can we assert the return value of `execute`?